### DISCUSSION TITLE
Add category when creating new helper

### DISCUSSION THREAD

**Discussion by AJediIAm** [home-assistant/frontend#21383] [Other feature requests]
**Use case:**
I was creating some helpers. After creating them, I needed to find them and set the category for each of them.
It would have been faster to choose the category when they are created. 

**Description:**
It's a fairly trivial change. I hope the mock-up below speaks for itself. 

New UI
![Create helper new](https://github.com/user-attachments/assets/e17dfde5-2d16-4aa5-bb7d-7413ffe0861b)

Old UI 
![Create helper](https://github.com/user-attachments/assets/5739e5b9-69ee-4e95-98f3-c284d453b8e7)

New workflow:
Step 1: Create new helper and assign category

Old workflow:
Step 1: Create new helper
Step 2: Find newly created helper in the ungrouped helpers
Step 3: Assign the category.

**Design considerations:**
A similar argument can be made for areas and labels. This would make the new helper menu a lot bigger. 
Currently, helpers can only be grouped by type and category. This might be a good argument to assign a category when creating a helper. 
If grouping by area is added in the future (which it should because of consistency with automations), assigning the area when creating a helper is an obvious addition. 
For now, adding the category would be good enough for me to speed up the workflow and promote organizing helpers.

**Comment by KNXBroker** [comment]
You have my vote, for automations it would be even more important! It is very annoying saving an automation and searching it again for categorisation.
